kitty/launch.py

```diff
@@ -81,7 +81,11 @@ def options_spec() -> str:
 --copy-env
 type=bool-set
 Copy the environment variables from the currently active window into the
-newly launched child process.
+newly launched child process. It will not copy environment variables that have
+been :code:`export`ed since the :code:`export` command does not change the environment
+of the shell, only of child processes launched in the shell. You can use remote control
+instead to have the current environment copied (:code:`allow_remote_control` must be on): 
+:code:`map kitty_mod+x remote_control launch --copy-env vim`.
```

Comment:
Hm, this isn't working for me, while running `kitty @launch --copy-env` in the shell works.

Reply:
It works for me though it fails 1 time out of 3 or each time I create a new tab. This is strange.

Reply:
I haven't managed to make it work each time and in every situation, I'll remove the text.